WordPressAuthenticator/Model/LoginFields.swift

```diff
@@ -166,7 +166,7 @@ public class LoginFieldsMeta: NSObject {
          socialService: SocialServiceName? = nil,
          socialServiceIDToken: String? = nil,
          googleUser: GIDGoogleUser? = nil,
-         appleUser: AppleUser? = nil) {
+         appleUser: SocialService.User? = nil) {
```

Comment:
Eventually, I think all these social related fields could be collapsed into one custom type to rule them all 💍